lib/commands/command-info.bash

```diff
@@ -2,9 +2,21 @@
 # shellcheck source=lib/functions/plugins.bash
 . "$(dirname "$(dirname "$0")")/lib/functions/plugins.bash"
 
+get_shell() {
+  if [[ -z $FISH_VERSION ]]; then
+    echo "fish"
+  elif [[ -z $ZSH_VERSION ]]; then
+    echo "zsh"
+  elif [[ -z $BASH_VERSION ]]; then
+    echo "bash"
+  else
+    echo $SHELL --version
```

Comment:
Currently the code invokes `$SHELL --version`, but this line actually just prints it, so we'd get `zsh --version` instead of `5.8`.

Reply:
ah good catch! addressed in [91527a3](https://github.com/asdf-vm/asdf/pull/1217/commits/91527a3df04b44a69df9de8a9789920ef42ecb0b)